tests/checks/normandy/test_remotesettings_recipes.py

```diff
@@ -65,24 +65,25 @@
         NORMANDY_URL.format(server=NORMANDY_SERVER, baseline_only=1),
         payload=[NORMANDY_RECIPE],
     )
-    mock_aioresponses.get(
-        REMOTESETTINGS_BASELINE_URL, payload={"data": [REMOTESETTINGS_RECIPE]}
-    )
     mock_aioresponses.get(
         REMOTESETTINGS_CAPABILITIES_URL,
         payload={"data": [REMOTESETTINGS_RECIPE, REMOTESETTINGS_RECIPE_WITH_CAPS]},
     )
+    mock_aioresponses.get(
+        REMOTESETTINGS_BASELINE_URL, payload={"data": [REMOTESETTINGS_RECIPE]}
+    )
```

Comment:
Just to confirm, moving these calls around doesn't change anything, right? It was part of the same commit so I wanted to make sure it didn't have a semantic meaning.

Reply:
No, I just wanted to have mocking in the same order as consuming